Assets/Scripts/GameSystems/TimeController.cs

```diff
@@ -143,21 +146,8 @@ private void OnDayComplete()
         daysComplete++;
     }
 
-    public TimeSpan CurrentTime()
```

Comment:
examples like this of the merge overwriting stuff from the last merged in. Might need to re pull and merge develop